lua/cmake-tools/const.lua

```diff
@@ -1,3 +1,11 @@
+local has_dap, dap = pcall(require, "dap")
+local dap_repl_open = nil
+
+if has_dap
+then
+  dap_repl_open = dap.repl.open
+end
+
```

Comment:
Not a lua dev so not sure if this the correct way to do this.

Reply:
Thanks, it's a way to fix it.